contrib/win32/win32compat/shell-host.c

```diff
@@ -39,6 +39,19 @@
 #include "misc_internal.h"
 #include "inc\utf.h"
 
+/* Terminal Types */
+#define TERM_TYPE_ANSI    "ansi"
+#define TERM_TYPE_VT100   "vt100"
+#define TERM_TYPE_XTERM   "xterm"
+
+#define TT_ANSI     0x0001        /* Codes in all ANSI like terminals. */
+#define TT_VT100    0x0002        /* VT100 */
+#define TT_XTERM    0x0004        /* Xterm */
+
+#define TTM_VT100X   (TT_VT100 | TT_XTERM)
```

Comment:
> win32-openssh client scrolling is not working with your current changes..

This can be observed only if you specify **TERM**=xterm
This should be the behavior in the absence of **TERM**: https://cloud.githubusercontent.com/assets/9350115/26391441/7f0a811a-406b-11e7-8465-941dde9720c7.gif